src/util/nodeSelectors.ts

```diff
@@ -124,6 +133,8 @@ export function argumentSelectionExtractor(): SelectionExtractor {
       node.type === ")" ||
       node.type === "[" ||
       node.type === "]" ||
+      node.type === ">" ||
+      node.type === "<" ||
```

Comment:
As above, I wonder if less-than or greater-than will be an issue here

Reply:
Good question…